src/runtime_src/core/pcie/emulation/hw_em/generic_pcie_hal2/xgq_hwemu.cxx

```diff
@@ -200,6 +201,7 @@ namespace hwemu {
   xgq_cmd::xgq_cmd()
   {
     cmdid = ++next_uid; //! Assign unique ID for each new command
+    ert_pkt = NULL;

```

Comment:
warning: use nullptr [modernize-use-nullptr]

```cpp
ert_pkt = NULL;
          ^~~~
          nullptr
```

Reply:
I knew this one. I was trying to follow 
 ert_pkt = NULL;
in https://github.com/Xilinx/XRT/blob/master/src/runtime_src/core/pcie/emulation/hw_em/generic_pcie_hal2/mbscheduler_hwemu.cxx#L41

But I am fixing it in xgq_hwemu now. Thanks.